src/app/Http/Controllers/CountryController.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
```suggestion
<?php
declare(strict_types=1);
```
Kiekvienas PHP failas turi prasidėti `<?php` ir `declare(strict_types=1);` antroje eilutėje. Tai yra PHP 7 versijos naujovė, kuri nurodo, jog privaloma laikytis griežto kintamųjų tipų laikymosi. Daugiau gali pasiskaityti čia: https://dev.to/robdwaller/how-php-type-declarations-actually-work-1mm5

Jei bus neaišku, sekantį kartą manęs paklausk ir paaiškinsiu išsamiau. Kol kas žinok tiek, kad būtina sudėti `declare(strict_types=1);` visų PHP (išskyrus BLADE ir konfigūracijas ir/arba `lang`) failų pradžioje.

Kad gyvenimas būtų lengvesnis, pakoreguok PHP Storm nustatymus `Ctrl + Alt + S` > `Editor` > `File and Code Templates` > `Files` > `PHP xxx` pridėdamas šią eilutę.